src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -0,0 +1,414 @@
+
+    return this;
+  }
+
+  private WebElement userOnChat(String userName, String message) {
+    return driver.findElement(By.cssSelector(String.format(message, userName)));
+  }
+
+  private WebElement userWelcomeMessage(String userName) {
+    return driver.findElement(By.cssSelector(String.format(USER_SELECTOR, userName)));
+  }
+
+  private WebElement userPostedMessage() {
+    return driver.findElement(By.xpath(String.format(PATH_MESSAGE_ON_CHAT, USER_MESSAGE_ON_CHAT)));
+  }
+
+  private WebElement userPrivateMessageNotification(int notificationCount) {
+    return driver.findElement(By.xpath(String.format(NOTIFICATION_COUNTER, notificationCount)));
+  }
+
+  private WebElement getUserUnbanMessage() {
+    return driver.findElement(By.xpath(String.format(USER_UNBAN_CONFIRM_MESSAGE, userName)));
+  }
+
+  private WebElement getUserUnbanLink(String userName) {
+   return driver.findElement(By.xpath(String.format(USER_UNBAN_LINK, userName)));
+  }
+
+  public boolean isUserOnChat() {
+    try {
+      return chatInlineAlert.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isMessageOnChat() {
+    try {
+      return userPostedMessage().isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isUserWelcomeMessageDisplayed(String userName) {
+    try {
+      return userWelcomeMessage(userName).isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageHeaderDispayed() {
+    try {
+      return privateMessagesHeader.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPermissionsErrorTitleDisplayed() {
+    try {
+      return permissionsErrorTitle.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageNotificationDisplayed() {
+    try {
+      return privateMessageNotification.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPricvateNotificationCountDisplayed(int notificationCount) {
+    try {
+      return userPrivateMessageNotification(notificationCount).isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isPrivateChatOpen() {
+    try {
+      return privateChatHeader.isDisplayed();
+    } catch (ElementNotFoundException e) {
+      return false;
+    }
+  }
+
+  public boolean isRegularUserDropdownDisplayed() {
+    if (userDropDownActionsElements.size() == REGULAR_USER_DROPDOWN_ELEMENTS) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  public boolean isUserInPrivateSectionDisplayed(String userName) {
+    try {
+      return userOnChat(userName, PRIVATE_MESSAGE_USER_SELECTOR).isDisplayed();
+    } catch (ElementNotFoundException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)